app/services/additional_claimants_csv/model_builder.rb

```diff
@@ -25,6 +26,13 @@ def attributes_from(row_data)
   end
 
   def sanitize(row_data)
+    check_csv_dob_format(row_data) if ATTRIBUTES.index(:date_of_birth)
```

Comment:
how about to use `.include?` instead of `.index` to make the statement easier to understand?